packages/db/src/meta/interface.ts

```diff
@@ -68,11 +68,10 @@ export const forAttachAndSchema = <C extends Collections>(options: {
   const serve = (serveOptions?: ConnectOptions<C>) => {
     const attachOptions = {
       adapter: (serveOptions || {}).adapter
-    }
+    };
     const workspace = attach(attachOptions);
 
     return new ApolloServer({
-      tracing: true,
```

Comment:
Wondering, what's the reason for removing this?  I should note -- I don't know actually know anything about Apollo server and so don't actually know what this does!  I guess my real question is, is removing this *part* of the v2 to v3 upgrade, or is it something you're doing *alongside* the v2 to v3 upgrade?

Reply:
No, you're right it's an apollo v3 thing. The v2 way is deprecated because the tool they used is no longer supported (https://www.apollographql.com/docs/apollo-server/api/plugin/inline-trace/). And, also, because this is a performance and usage reporting tool it should not be enabled by default and should be configurable or provided by an ENV var at such time when it is in a dev/testing/staging env or in production if faced with prod-specific issues and the benchmarking/reporting is useful. 

If it was being used for an optimization in dev it may be a little premature for that. The old v2 was used by GraphQL playground but it is unclear whether v3's inline trace plugin is as well. We haven't lost the playground in v3 but i havent checked if they've replaced or navigated without it.

Reply:
OK, good to know, thanks for clarifying!